brave/brave-core
# Move wallet functions into separate files

Resolves https://github.com/brave/brave-browser/issues/4913

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- make sure that you can create wallet
- make sure that wallet recovery works

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
